test/common/cache_utils.py

```diff
@@ -0,0 +1,46 @@
+import os
+import json
+import torchtext
+from .parameterized_utils import load_params
+
+CACHE_STATUS_FILE = '.data/cache_status_file.json'
```

Comment:
Eventually instead of pass/fail we could include the hash of the expected files to prevent a cache corruption between creation and PR consumption

Reply:
Added a follow-up task